src/Nethermind/Nethermind.Merge.Plugin/GC/IGCStrategy.cs

```diff
@@ -0,0 +1,10 @@
+// SPDX-FileCopyrightText: 2023 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+namespace Nethermind.Merge.Plugin.GC;
+
+public interface IGCStrategy
+{
+    bool ShouldTryToPreventGCDuringBlockProcessing();
+    int ShouldForceGCBetweenBlockProcessing();
```

Comment:
```suggestion
    int MaxGenerationToGCCollectBetweenBlocks {get; }
```

it would work much better for me as `int` should not be `Should` I believe